tools/appliance/definitions/systemvmtemplate/install_systemvm_packages.sh

```diff
@@ -71,7 +71,7 @@ function install_packages() {
     iptables-persistent \
     libtcnative-1 libssl-dev libapr1-dev \
     python-flask \
-    haproxy \
+    haproxy quagga \
```

Comment:
Thanks @abhinandanprateek I'll see what I can do about building the systemvmtemplates